okhttp/src/test/java/okhttp3/CallTest.java

```diff
@@ -963,8 +971,7 @@ private void postBodyRetransmittedAfterAuthorizationFail(String body) throws Exc
         .url(server.url("/"))
         .build();
     executeSynchronously(request)
-        .assertCode(200)
-        .assertBody("abc");
+        .assertFailure(SocketException.class);
```

Comment:
@swankjesse this is an explicit change in behaviour

Reply:
Totally fine. In either case the caller is probably not going to catch the unchecked exception in the interceptor.